### ISSUE TITLE
GitHub API calls

### ISSUE THREAD

**Issue by uk1988** [grafana/loki#14643]
Hej All

We have been utilizing the GitHub API to retrieve build packages for our projects, using the following CURL command for downloads:
`curl -sfSL --retry 2 --header "Accept: application/json" "https://api.github.com/repos/grafana/loki/releases"`

Recently, we observed that releases belonging to the v2.7.x series are no longer accessible through the API. Conversely, these releases are still available for download directly via the webpage: https://github.com/grafana/loki/releases/tag/v2.7.7

We are trying to ascertain whether this discrepancy results from limitations associated with the public API or if there has been a deliberate decision to remove older versions from the API endpoints.

Best Regards,
Uros

**Comment by paul1r** [comment]
I believe this is due to the [default](https://docs.github.com/en/rest/releases/releases?apiVersion=2022-11-28) of 30 items per page in the API.

Running `curl -sfSL --retry 2 --header "Accept: application/json" "https://api.github.com/repos/grafana/loki/releases?per_page=100" ` yielded items in the 2.7 release timeframe

**Comment by uk1988** [comment]
yes, that was it, guess we did not look the GitHub API close enough 
thank you!
